buildSrc/src/main/java/PosthogBuildConfig.kt

```diff
@@ -47,7 +47,7 @@ object PosthogBuildConfig {
 
     object Dependencies {
         // runtime
-        val LIFECYCLE = "2.7.0"
+        val LIFECYCLE = "2.6.2"
```

Comment:
2.7.0 is compiled against API 34 which is incompatible with older versions, unless the final app also uses API 34